src/dbtools/Dockerfile

```diff
@@ -4,23 +4,25 @@
 # 
 
 ##### Oracle Instant Client version
-ARG clientVersion=19.5
+ARG clientVersion=21
```

Comment:
![21](https://media0.giphy.com/media/xzRpQnFfrdVRu/giphy.gif)

Reply:
DB team changed the strategy, just following vibe :)